core/indexer/elasticClient.go

```diff
@@ -50,12 +50,12 @@ func (ec *elasticClient) CheckAndCreateTemplate(templateName string, template *b
 }
 
 // CheckAndCreatePolicy creates a new index policy if it does not already exist
-func (ec *elasticClient) CheckAndCreatePolicy(policyName string, policy *bytes.Buffer) error {
-	if ec.PolicyExists(policyName) {
+func (ec *elasticClient) CheckAndCreatePolicy(policyName string, policy *bytes.Buffer, kibanaURL string) error {
+	if ec.PolicyExists(policyName, kibanaURL) {
```

Comment:
I would have the base url for kibana inside `elasticClient` since this component does the actual requests, instead of passing it to every function that needs it

Reply:
done